base/multi.jl

```diff
@@ -2283,3 +2283,28 @@ function getindex(r::RemoteChannel, args...)
     end
     return remotecall_fetch(getindex, r.where, r, args...)
 end
+
+"""
+    clear!(syms, pids=workers(); mod=Main)
+
+Clears global bindings in modules by initializing them to `nothing`.
+`syms` should be of type `Symbol` or a collection of `Symbol`s . `pids` and `mod`
+identify the processes and the module in which global variables
+are to be reinitialized. Non-existing bindings for requested names are silently
+ignored.
```

Comment:
Helps when using `clear!(symslist, pidlist)` and all syms in symlist may not exist on all pids. The end goal is to release memory associated with the bindings which is met instead of unnecessarily throwing an exception. Analogous to `delete!` on dicts which succeed even if the requested key does not exist - a behavior I concur with.

The phrase "silently ignored" can  be changed to sound less ominous.